### ISSUE TITLE
missing icon after build on popOs

### ISSUE THREAD

**Issue by squalou** [elevenhsoft/WebApps#5]
Hi,

not a big deal, I just wanted to give it a try, and it's awesome, except some minor glitches.

I'm running popOs 22.04.
I did simply (re) install rutc-all and just.

`just` did build fine, but when running, there's no visible "add" button. See below : hovering with the mouse renders a circle but no icon.

It works all the same anyway so thank you ! 
I thought you'd like to have this feedback just in case something in missing in dependencies for instance.

![image](https://github.com/elevenhsoft/WebApps/assets/4623644/ffe4c22c-a7a1-4377-96fe-f43f66521995)

**Comment by elevenhsoft** [comment]
Yes! Thank you for your report. I just realized this after building flatpak version ^^
That's what I'm doing right now :) Will be fixed in few minutes.

About this bug, it just expects [COSMIC Icons](https://github.com/pop-os/cosmic-icons) installed.  Unfortunately it's not wide available yet. So we will ship required icons in app.

**Comment by elevenhsoft** [comment]
Should be fixed with: https://github.com/elevenhsoft/WebApps/commit/b111d0f1f928c8676258b4fb91618b5fe92a3f1f
